### ISSUE TITLE
Full screen merchant center set up screen/page

### ISSUE THREAD

**Issue by findingsimple** [woocommerce/google-listings-and-ads#44]
**Description:**

Set up basic scaffolding for the merchant center set up screen/page

Upon clicking the "Get started" (or "Continue Setup") on the [getting started page](https://github.com/woocommerce/google-for-woocommerce/issues/39) a full-page set up screen/page should be displayed

At the top left of the page will be a backlink with the text "< Get started with Google for eCommerce" -  this will return the user to the [getting started page](https://github.com/woocommerce/google-for-woocommerce/issues/39)

At the top right of the page will be a help link pointing to extension documentation (location to be confirmed).

The set up process will consist of 2 steps:

1. Set up your accounts
2. Configure your free listing

Each of these is essentially a tabbed content area (or equivalent approach).

- Start at Step 1.
- Step 2 is only available once Step 1 is completed (i.e. all accounts are connected/created).
- Step2 text etc. is not clickable (disabled) until step 1 is completed
- Once "Set up your accounts" is complete the number "1" becomes a blue checkmark and content from step 2 is displayed. it becomes clickable so that merchants can go back to step 1 if they wish (and vice versa). If they then disconnect an account I guess step 2 becomes unavailable again.
- At the top of each step is a label (Step 1, Step2), Heading (Set up your accounts), and a brief description of the step.
- On step 2 there is a mock product listing below the heading and description - for initial implementation use a placeholder image.
- Continue button at the bottom right is disabled until the steps are completed.

**Note** - during development, we may need to fake some of this logic initially (or options etc.) to avoid being blocked - actual functionality can be added as it is finalised.

---

**Screenshots**

Step 1

![Screen Shot 2020-11-03 at 11 03 08 am](https://user-images.githubusercontent.com/355014/97933996-4d250780-1dc4-11eb-8775-2fa47850084d.png)

Step 2

![Screen Shot 2020-11-03 at 11 02 49 am](https://user-images.githubusercontent.com/355014/97933966-354d8380-1dc4-11eb-9584-727cece77f77.png)

---

**Technical**

* Button component should be used for (for example) the continue button
* We have a later flow/epic for [Ads Account set up/creation](https://github.com/woocommerce/google-for-woocommerce/issues/20) which would use a similar full page view - just something to be considered
* Will need to be able to "link" to step 1 when redirecting after authentication
* "See other listings layouts" link is still TBC

---

**Acceptance criteria:**

* [ ] Logic above is followed
* [x] Button colors updates to reflect admin profile color scheme
* [x] Tracks event fired on clicking the back link, help button or continue button
* [x] Translatable text
* [ ] SVG for images provided size is acceptable.

---

**Dependencies**

* [Getting started page](https://github.com/woocommerce/google-for-woocommerce/issues/39)

**Out of bounds/rabbit holes**

* For the Initial implementation lets just use a static placeholder image for the mock product listing on step 2- we'll look at dynamically generating mock product listings based on store products as a nice to have in a follow-up issue.
* Looking into the more complicated elements of the setup screen e.g. connect to wordpress, connect to google etc. etc. should be handled in separate PRs/issues

**Event tracking:**

* [x] Tracks event fired on clicking the back link, help button or continue button
